.jshintrc

```diff
@@ -20,6 +20,9 @@
   "worker": true,
   "globals": {
     "JSON": false,
-    "URLSearchParams": false
+    "URLSearchParams": false,
+    "define": true,
+    "module": true,
+    "global": true
```

Comment:
Sadly, I didn't find any. There are a lot of things like that in eslint but I cannot find those for jshint.